src/features/controls.ts

```diff
@@ -366,6 +373,10 @@ export const ControlsMixin = (ModelViewerElement:
           this[$updateAria]();
           this[$needsRender]();
         }
+
+        [$onOrbigChange]() {
+          this.dispatchEvent(new CustomEvent('user-camera-orbit-change'));
```

Comment:
It might be nice to create a PR which moves all these public event's strings to its own module.